hook/hookstest/stub.go

```diff
@@ -22,6 +23,20 @@ type Stub struct {
 
 var _ hook.Points = (*Stub)(nil)
 
+// NewBlock calls [types.NewBlock] with its arguments.
+func (*Stub) NewBlock(
+	header *types.Header,
+	txs []*types.Transaction,
+	receipts []*types.Receipt,
+) *types.Block {
+	return types.NewBlock(header, txs, nil, receipts, trie.NewStackTrie(nil))
```

Comment:
```suggestion
	return types.NewBlock(header, txs, nil, receipts, saetest.TrieHasher())
```

Although it's semantically identical, it stops the reader having to think about the appropriateness of `trie.NewStackTrie(nil)` here because it's really weird IMO.